lib/apartmentex.ex

```diff
@@ -1,24 +1,18 @@
 defmodule Apartmentex do
-
   @schema_prefix Application.get_env(:apartmentex, :schema_prefix) || "tenant_"
   @tenant_migration_folder "priv/repo/tenant_migrations"
   #warning: don't change this after you already have tenants
 
-
-  alias Ecto.Repo.Queryable
-  alias Ecto.Repo.Model
   alias Ecto.Changeset
-  alias Apartmentex.Helpers.Queryable, as: QHelpers
-  alias Apartmentex.Helpers.Model, as: MHelpers
-  require Ecto.Query
 
   def new_tenant(repo, tenant) do
     prefix = build_prefix(tenant)
     case repo.__adapter__ do
       Ecto.Adapters.Postgres -> Ecto.Adapters.SQL.query(repo, "CREATE SCHEMA #{prefix}", [])
       Ecto.Adapters.MySQL -> Ecto.Adapters.SQL.query(repo, "CREATE DATABASE #{prefix}", [])
     end
-    Apartmentex.Migrator.run(repo, @tenant_migration_folder, :up, [all: true, prefix: String.to_atom(prefix)])
+    Ecto.Migration.Supervisor.start_link
+    Ecto.Migrator.run(repo, @tenant_migration_folder, :up, [all: true, prefix: String.to_atom(prefix)])
```

Comment:
@Dania02525 since the `:ecto` application has been added to the apartmentex `mix.exs` file, is there still a need to call `Application.ensure_all_started(:ecto)` for tests?